src/fonts/lib/Space.ttx

```diff
@@ -110,46 +108,6 @@
   </OS_2>
 
   <name>
-    <namerecord nameID="0" platformID="1" platEncID="0" langID="0x0" unicode="True">
-      Copyright (c) 2009-2010 Design Science, Inc.
-Copyright (c) 2014-2018 Khan Academy
-    </namerecord>
```

Comment:
`platformID="1"` is the Macintosh platform which is why we're getting rid of all of these.